src/components/StateListItem.vue

```diff
@@ -56,6 +67,10 @@
 </script>
 
 <style>
+.state-list-item .p-list-item-input__checkbox { @apply
+  bg-transparent
```

Comment:
As far as I've tested this is the only real change in function/behavior for this component and overall should be a strict improvement; previously if a state was passed that wasn't one we could map (as could be the case if states are deleted or new ones are introduced ahead of the UI mapping utilities), the styling for this component would fall back to the base prefect-design version, which has a background not dissimilar to "cancelled" and "crashed" states in most CVD palettes. This causes the stateful background to instead show as transparent, allowing the component to feel "stateless" instead